internal/driver/webhtml.go

```diff
@@ -1329,7 +1329,7 @@ function viewer(baseUrl, nodes) {
   <script>viewer(new URL(window.location.href), {{.Nodes}});</script>
   <script>{{template "d3flamegraphscript" .}}</script>
   <script>
-    var data = {{.FlameGraph}};
+    var data = JSON.parse("{{.FlameGraph}}");
```

Comment:
Maybe you could add a comment indicating why we're calling JSON.parse here? Basically a short summary of what you have in the PR describe?

Reply:
+1

Reply:
Thanks for adding the comment, @vilterp 👍